### PR TITLE
modify bug: SNAT mode, ipvsadm displaying and editing the SVC will fail.

### PR DESCRIPTION
**Pull request by yfming**
example：
ipvsadm -ln -H "snat-svc"
ipvsadm -E -H "snat-svc"

### CODE REVIEW

File: tools/keepalived/keepalived/check/libipvs.c [hunk 1]
```diff
@@ -704,10 +703,17 @@ ipvs_get_service(ipvs_service_t *hint, lcoreid_t cid)
...
+	dpvs_svc.user.proto = hint->user.protocol;
+	memcpy(&dpvs_svc.user.addr, &hint->nf_addr, sizeof(dpvs_svc.user.addr));
+	dpvs_svc.user.port = hint->user.port;
+	dpvs_svc.user.fwmark = hint->user.fwmark;
+	dpvs_svc.user.flags = hint->user.flags;
+	dpvs_svc.user.netmask = hint->user.netmask;
+	snprintf(dpvs_svc.user.srange, sizeof(dpvs_svc.user.srange), "%s", hint->user.srange);
+	snprintf(dpvs_svc.user.drange, sizeof(dpvs_svc.user.drange), "%s", hint->user.drange);
+	snprintf(dpvs_svc.user.iifname, sizeof(dpvs_svc.user.iifname), "%s", hint->user.iifname);
+	snprintf(dpvs_svc.user.oifname, sizeof(dpvs_svc.user.oifname), "%s", hint->user.oifname);
```
Comment (ywc689) [inline tools/keepalived/keepalived/check/libipvs.c:716]: I think the macro `IPVS_2_DPVS` also supports type `dpvs_service_entry_t`. Please make use of it to simplify the codes.

Comment (yfming) [reply tools/keepalived/keepalived/check/libipvs.c]: already simplify the codes

**Review by ywc689 — APPROVED** [review]
Please merge the commits (command `rebase -i` may help), and let commit message more expressive.

**Comment by yfming** [comment]
> Please merge the commits (command `rebase -i` may help), and let commit message more expressive.

已修改提交信息

**Pull request merged**